.gitignore

```diff
@@ -4,6 +4,7 @@
 *.xml
 !/tests/**/fixtures/*.xml
 
+.vscode/
```

Comment:
duplicated folder?

Reply:
Not sure what you mean, I've checked out the repository, opened it up using vs code which creates the folder. I don't see duplicates here. Did I setup my environment incorrectly?